### PR TITLE
Prevent a transfer item’s expected quantity from being updated when its stock transfer is finalized

### PR DESCRIPTION
**Pull request by richardnuno**

### CODE REVIEW

File: core/app/models/spree/transfer_item.rb [hunk 1]
```diff
@@ -31,6 +32,13 @@ def ensure_stock_transfer_not_finalized
       end
     end
 
+    def prevent_expected_quantity_update_stock_transfer_finalized
+      if changed.include?("expected_quantity") && !stock_transfer.finalizable?
```
Comment (jhawthorn) [inline core/app/models/spree/transfer_item.rb]: Is this the same as `expected_quantity_changed?`

Comment (richardnuno) [inline core/app/models/spree/transfer_item.rb]: It is, good catch.

**Comment by richardnuno** [comment]
@jhawthorn updated to use `expected_quantity_changed?`. Good otherwise?

**Comment by richardnuno** [comment]
@jhawthorn sorry, had missed the second part of the comment. That's a good point, the expected_quantity shouldn't be changed no matter what the shipping or close states are. As long as it's finalized it shouldn't be updated. Will make that change.

**Comment by richardnuno** [comment]
@jhawthorn updated with both suggestions now.

**Comment by gmacdougall** [comment]
:+1:

**Comment by athal7** [comment]
:+1:

**Pull request merged**